### PR TITLE
Log levels

### LINKED ISSUE #1069: Make logging levels configurable
Add environment variables to override the logging levels in our application.  We may need several different variables in order to cover all the Akka and Scala logs.

### PR DESCRIPTION
**Pull request by jmarin**
Allows configuring log level for the `HMDA Platform`, controlled by the `HMDA_LOGLEVEL` environment variable

Closes #1069

### CODE REVIEW

**Comment by schbetsy** [comment]
Closes #1069?

File: validation/src/main/resources/application.conf [hunk 1]
```diff
@@ -1,6 +1,7 @@
 akka {
-  stdout-loglevel = "DEBUG"
-  loglevel = "OFF"
+  stdout-loglevel = "OFF"
```
Comment (hkeeler) [inline validation/src/main/resources/application.conf]: Do we want to turn this to `OFF`?  Sounds like we'll lose startup logs if we do this:
* http://doc.akka.io/docs/akka/2.5.3/scala/logging.html#logging-to-stdout-during-startup-and-shutdown

Comment (jmarin) [inline validation/src/main/resources/application.conf]: Not exactly. This is silenced only when the logs are being sent to a file. When logging to a file, these startup logs show up at the console always. In any case, I can see the startup logs just fine

Comment (hkeeler) [inline validation/src/main/resources/application.conf]: OK, cool.

File: api/src/main/scala/hmda/api/HmdaPlatform.scala [hunk 2]
```diff
@@ -33,6 +33,19 @@ object HmdaPlatform {
...
+    log.info(
+      """
+        | #     # #     # ######     #       ######
+        | #     # ##   ## #     #   # #      #     # #        ##   ##### ######  ####  #####  #    #
+        | #     # # # # # #     #  #   #     #     # #       #  #    #   #      #    # #    # ##  ##
+        | ####### #  #  # #     # #     #    ######  #      #    #   #   #####  #    # #    # # ## #
+        | #     # #     # #     # #######    #       #      ######   #   #      #    # #####  #    #
+        | #     # #     # #     # #     #    #       #      #    #   #   #      #    # #   #  #    #
+        | #     # #     # ######  #     #    #       ###### #    #   #   #       ####  #    # #    #
+        |
```
Comment (schbetsy) [inline api/src/main/scala/hmda/api/HmdaPlatform.scala]: oh... in docker compose, this happened:

```
api_1       |   #     # #     # ######     #       ######
ui_1        |  2017/08/24 14:00:38 [warn] 10#10: duplicate MIME type "text/html" in /etc/nginx/nginx.conf:44
lb_1        |    acl host_rule_6 hdr_reg(host) -i ^.*$
api_1       |   #     # ##   ## #     #   # #      #     # #        ##   ##### ######  ####  #####  #    #
ui_1        |  nginx: [warn] duplicate MIME type "text/html" in /etc/nginx/nginx.conf:44
mail_dev_1  |  MailDev app running at 0.0.0.0:80
lb_1        |    acl host_rule_6_port hdr_reg(host) -i ^.*:443$
ui_1        |  2017/08/24 14:00:38 [notice] 10#10: using the "epoll" event method
mail_dev_1  |  MailDev SMTP Server running at 0.0.0.0:25
api_1       |   #     # # # # # #     #  #   #     #     # #       #  #    #   #      #    # #    # ##  ##
lb_1        |    acl path_rule_6 path_reg -i ^/.*$
ui_1        |  2017/08/24 14:00:38 [notice] 10#10: nginx/1.10.3
api_1       |   ####### #  #  # #     # #     #    ######  #      #    #   #   #####  #    # #    # # ## #
ui_1        |  2017/08/24 14:00:38 [notice] 10#10: built by gcc 4.9.2 (Debian 4.9.2-10) 
lb_1        |    use_backend SERVICE_hmdaplatform_ui if path_rule_6 host_rule_6 or path_rule_6 host_rule_6_port
api_1       |   #     # #     # #     # #######    #       #      ######   #   #      #    # #####  #    #
ui_1        |  2017/08/24 14:00:38 [notice] 10#10: OS: Linux 4.4.74-boot2docker
lb_1        |  frontend port_80
api_1       |   #     # #     # #     # #     #    #       #      #    #   #   #      #    # #   #  #    #
lb_1        |    bind :80
api_1       |   #     # #     # ######  #     #    #       ###### #    #   #   #       ####  #    # #    # 

```

**Comment by jmarin** [comment]
I've added a link to where the available logging levels for Akka come from. For more information on how logging works in `Akka`, see [here](http://doc.akka.io/docs/akka/2.5.3/scala/logging.html) (I don't think including documentation links in the README makes sense, as they change with software version).

**Pull request merged**